ci-operator/step-registry/quay/quay-tests/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
@dongboyan77 it means you are going to create steps like `quay-quay-tests-...` which may be not what you want.
Can you add some job config in this PR?